Packet++/src/HttpLayer.cpp

```diff
@@ -857,14 +857,12 @@ HttpResponseStatusCode HttpResponseFirstLine::parseStatusCode(const char* data,
 
 	std::string codeString = std::string(data + 9, 3);
 
-	if(!codeString.empty() && std::find_if(codeString.begin(), codeString.end(), [](unsigned char c){ return !std::isdigit(c); }) == codeString.end())
+	if(!codeString.empty() || std::find_if(codeString.begin(), codeString.end(), [](unsigned char c){ return !std::isdigit(c); }) == codeString.end())
```

Comment:
You should remove the `!` in `!codeString.empty()`

Reply:
actually, I found that the original version is correct ...
`!codeString.empty() && std::find_if(codeString.begin(), codeString.end(), [](unsigned char c){ return !std::isdigit(c); }) == codeString.end()`
or 
`codeString.empty() || std::find_if(codeString.begin(), codeString.end(), [](unsigned char c){ return !std::isdigit(c); }) != codeString.end()`

are the same

Reply:
yes you're right, sorry about that...

I personally find using `||` a bit more clear, but maybe it's just me